worker-sys/src/ext/asset_handler.rs

```diff
@@ -0,0 +1,13 @@
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(module = "__STATIC_CONTENT_MANIFEST")]
+extern "C" {
```

Comment:
We should try to avoid having anything but wasm-bindgen glue in the `worker-sys` crate if possible, this global should be moved to the `worker` crate.